app/src/components/common/icons/IconSettings.tsx

```diff
@@ -1,12 +1,43 @@
 import React from 'react'
+import styled from 'styled-components'
 
-import theme from '../../../theme'
+const Wrapper = styled.svg<{ hoverEffect: boolean | undefined }>`
```

Comment:
you don't need or undefined since below when you put hoverEffect=false that means that false is default value if not present...